Lib/fontmake/__main__.py

```diff
@@ -438,6 +440,31 @@ def main(args=None):
     glyphs_path = args.pop("glyphs_path")
     ufo_paths = args.pop("ufo_paths")
     designspace_path = args.pop("mm_designspace")
+    any_input = glyphs_path or ufo_paths or designspace_path
+
+    for file in unknown:
+        if any_input and not (ufo_paths and file.endswith(".ufo")):
+            logging.warn(
+                "Only one input file type accepted; additional file '%s' ignored" % file
+            )
+        elif file.endswith(".glyphs"):
+            glyphs_path = file
+            any_input = True
+        elif file.endswith(".designspace"):
+            designspace_path = file
```

Comment:
-g/-m/-u are mutually exclusive (enforced by argparse), whereas this seems to allow specifying e.g. _both_ e.g. a .glyphs and a .designspace file, I don't think we want to allow that in the same fontmake invocation. I suggest checking that mutual exclusivity of input type is still the case

Reply:
oh I just realised what you're doing now, you are ignoring additional input file types and only taking the first one with a warning. I would probably let it blow up with an error instead, similarly to specifying both `-g` and `-m` in the same invokation